apiserver/facades/agent/keyupdater/authorisedkeys.go

```diff
@@ -27,11 +34,12 @@ type KeyUpdater interface {
 // KeyUpdaterAPI implements the KeyUpdater interface and is the concrete
 // implementation of the api end point.
 type KeyUpdaterAPI struct {
-	state      *state.State
-	model      *state.Model
-	resources  facade.Resources
-	authorizer facade.Authorizer
-	getCanRead common.GetAuthFunc
+	controllerConfigService ControllerConfigService
+	state                   *state.State
+	model                   *state.Model
+	resources               facade.Resources
+	authorizer              facade.Authorizer
+	getCanRead              common.GetAuthFunc
 }
 
 var _ KeyUpdater = (*KeyUpdaterAPI)(nil)
```

Comment:
The comments in the methods `func (api *KeyUpdaterAPI) WatchAuthorisedKeys()` and `func (api *KeyUpdaterAPI) AuthorisedKeys()` should be changed: `... stored in the model and controller config.`